src/main/resources/data/createaddition/recipes/crafting/straw.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Would like this to be the same rolling recipe as on forge

Reply:
this applies to the 1.19 branch too

Reply:
Ye i'll fix bc it was just a place holder